src/TestTrait.php

```diff
@@ -36,25 +44,58 @@ public static function erisSetupBeforeClass()
      */
     public function erisSetup()
     {
-        $this->seedingRandomNumberGeneration();
-        $this->minimumEvaluationRatio(0.5);
+        $this->seed = intval(getenv('ERIS_SEED') ?: (microtime(true)*1000000));
```

Comment:
Was this inlined to avoid making it available where the trait is used? Would still have it as a separate method annotated with `@internal` for clarity